### ISSUE TITLE
SysOID for generic Linux server is matched to physical Dell server

### ISSUE THREAD

**Issue by dgcom** [kentik/snmp-profiles#36]
Any generic Linux server with netSNMP agent will return this SysOID: [https://oidref.com/1.3.6.1.4.1.8072.3.2.10](https://oidref.com/1.3.6.1.4.1.8072.3.2.10)
This is currently matching dell-poweredge.yml which describes physical Dell server which is not correct - this can be VM, or any generic hardware.
This profile should really pull info from generic MIBs which would exist on default plain vanilla Linux install.
And the other question is - how this will be represented in NewRelic? Ideally, this should be able to pull most info needed to represent real infrastructure host.
Currently, this fails to pull any useful counter into NewRelic at all.

**Comment by thezackm** [comment]
closed with PR #72

**Comment by dgcom** [comment]
Interesting... I was actually about to post the profile I have used for generic Linux host with fully configured NetSMNP agent - see below.
I am still not sure if kentik-net-snmp will be recognized as host in NewRelic (it probably should).

```
# Generic profile for Linux running NetSNMP agent
---
extends:
  - system-mib.yml
  - if-mib.yml
  - ip-mib.yml
  - tcp-mib.yml
  - udp-mib.yml
  - ucd-mib.yml
  - host-resources-mib.yml

provider: kentik-linux-netsnmp

# disable the SNMP bulk walk operation for these devices
no_use_bulkwalkall: true

# SNMPv2-MIB::sysObjectID.0 = OID: NET-SNMP-MIB::netSnmpAgentOIDs.10
sysobjectid:
  - 1.3.6.1.4.1.8072.3.2.10

metrics:
# system mib
# There are more SNMP metrics available
  - MIB: SNMPv2-MIB
    symbol:
      OID: 1.3.6.1.2.1.11.29.0
      name: snmpOutTraps
# IF-MIB - good coverage
# IP-MIB - captures tables, may need some additional scalars, ex. ipOutNoRoutes
# TCP-MIB - connection table can be useful
# UDP-MIB - udpHCInDatagrams/udpHCOutDatagrams - may need to be replaced with udpInDatagrams/udpOutDatagrams
# UCD-SNMP-MIB and UCD-DISKIO-MIB - good coverage
# HOST-RESOURCES-MIB - possible add: hrMemorySize
